hack/testing/logging.sh

```diff
@@ -137,203 +139,12 @@ os::cmd::expect_success 'oadm router'
-    os::cmd::try_until_text "oc get dc -o name -l component=es-ops" "ops"
-    ops_dc=$(oc get dc -o name -l component=es-ops) || exit 1
-    os::cmd::expect_success "oc patch $ops_dc \
-       -p '{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"elasticsearch\",\"securityContext\":{\"privileged\": true}}]}}}}'"
-
-    os::cmd::try_until_text "oc deploy $ops_dc --latest" "Started" "$(( 3 * TIME_MIN ))"
-fi
-# see if expected pods are running
-os::cmd::try_until_text "oc get pods -l component=es" "Running" "$(( 3 * TIME_MIN ))"
-os::cmd::try_until_text "oc get pods -l component=kibana" "Running" "$(( 3 * TIME_MIN ))"
-os::cmd::try_until_text "oc get pods -l component=curator" "Running" "$(( 3 * TIME_MIN ))"
-if [ "$ENABLE_OPS_CLUSTER" = "true" ] ; then
-    # make sure the expected PVC was created and bound
-    os::cmd::try_until_text "oc get persistentvolumeclaim es-ops-pvc-1" "Bound" "$(( 1 * TIME_MIN ))"
-    # make sure the expected pods are running
-    os::cmd::try_until_text "oc get pods -l component=es-ops" "Running" "$(( 3 * TIME_MIN ))"
-    os::cmd::try_until_text "oc get pods -l component=kibana-ops" "Running" "$(( 3 * TIME_MIN ))"
-    os::cmd::try_until_text "oc get pods -l component=curator-ops" "Running" "$(( 3 * TIME_MIN ))"
-fi
-
-if [ -n "$ES_VOLUME" ] ; then
-    if [ ! -d $ES_VOLUME ] ; then
-        sudo mkdir -p $ES_VOLUME
-        sudo chown 1000:1000 $ES_VOLUME
-    fi
-    # allow es and es-ops to mount volumes from the host
-    os::cmd::expect_success "oadm policy add-scc-to-user hostmount-anyuid \
-         system:serviceaccount:logging:aggregated-logging-elasticsearch"
-    # get es dc
-    esdc=`oc get dc -l component=es -o jsonpath='{.items[0].metadata.name}'`
-    # shutdown es
-    espod=`get_running_pod es`
-    os::cmd::expect_success "oc scale dc $esdc --replicas=0"
-    os::cmd::try_until_failure "oc describe pod $espod > /dev/null" "$(( 3 * TIME_MIN ))"
-    # mount volume manually on ordinary cluster
-    os::cmd::expect_success "oc volume dc/$esdc \
-                             --add --overwrite --name=elasticsearch-storage \
-                             --type=hostPath --path=$ES_VOLUME"
-    # start up es
-    os::cmd::try_until_text "oc deploy $esdc --latest" "Started" "$(( 3 * TIME_MIN ))"
-    os::cmd::expect_success "oc scale dc $esdc --replicas=1"
-    os::cmd::try_until_text "oc get pods -l component=es" "Running" "$(( 3 * TIME_MIN ))"
-fi
-
-# start fluentd
-os::cmd::try_until_success "oc get daemonset logging-fluentd" "$(( 1 * TIME_MIN ))"
-os::cmd::expect_success "oc label node --all logging-infra-fluentd=true"
-
-# the old way with dc's
-# # scale up a fluentd pod
-# os::cmd::try_until_success "oc get dc logging-fluentd"
-# os::cmd::expect_success "oc scale dc logging-fluentd --replicas=1"
 
-os::cmd::try_until_text "oc get pods -l component=fluentd" "Running" "$(( 5 * TIME_MIN ))"
-### logging component pods are now created and deployed ###
+#initialize logging stack
```

Comment:
can you add a space after the # just so its more obvious this is a doc comment and not some commented out code?